CryptoTechReminderSystem.Test/Gateway/SlackGatewayTests.cs

```diff
@@ -22,18 +22,64 @@ public class SlackGatewayTests
         [TestFixture]
         public class CanSendMessage
         {
-            private FluentSimulator _slackApi;
+            private SlackSimulator _slackApi;
             private SlackGateway _slackGateway;
             
+            class SlackSimulator
+            {
+                private FluentSimulator _simulator;
+                public bool SpyMethodCalled;
+                public string ReceivedErrorMessage;
+
+                public SlackSimulator(string address)
+                {
+                    _simulator = new FluentSimulator(address);
+                    SpyMethodCalled = false;
+                }
+                
+                public ReceivedRequest[] ReceivedRequests()
+                {
+                    return _simulator.ReceivedRequests.ToArray();
+                }
+
+                public void ShouldRespondWithOk()
```

Comment:
This can just be named `RespondWithOk`, I think the `Should` is unnecessary

Reply:
I agree, resolved in : https://github.com/chubberlisk/crypto_tech/pull/37/commits/853dd5704b4410861457d85ccbebdec371e417f2